include/certifier_framework.h

```diff
@@ -42,35 +43,71 @@ namespace certifier {
       key_message policy_key_;
       string encryption_algorithm_;
 
+      // -------------------------------------------------------------------
+      // Different types of artifacts that are tracked in the Policy Store
+      // -------------------------------------------------------------------
+      //
+      trusted_service_message** ts_;
       int max_num_ts_;
       int num_ts_;
-      trusted_service_message** ts_;
+
+      tagged_signed_claim** tsc_;
       int max_num_tsc_;
       int num_tsc_;
-      tagged_signed_claim** tsc_;
+
+      storage_info_message** si_;
       int max_num_si_;
       int num_si_;
-      storage_info_message** si_;
+
+      tagged_claim** tc_;
       int max_num_tc_;
       int num_tc_;
-      tagged_claim** tc_;
+
+      channel_key_message** tkm_;
       int max_num_tkm_;
       int num_tkm_;
-      channel_key_message** tkm_;
+
+      tagged_blob_message** tagged_blob_;
       int max_num_blobs_;
       int num_blobs_;
-      tagged_blob_message** tagged_blob_;
 
     public:
 
+      // Default capacity for artifacts in store is MAX_NUM_ENTRIES for
+      // each type of artifact defined above.
       policy_store();
-      policy_store(const string enc_alg, int max_trusted_services, int max_trusted_signed_claims,
-          int max_storage_infos, int max_claims, int max_keys, int max_blobs);
+
+      // Initialize a Policy Store for an encryption algorithm specifying
+      // the capacity of the store in terms of max-numbers of artifacts
+      // that can be tracked.
+      policy_store(const string enc_alg, int max_trusted_services,
+                   int max_trusted_signed_claims, int max_storage_infos,
+                   int max_claims, int max_keys, int max_blobs);
```

Comment:
A consistent pattern of diffs in this change set is to align the parameters to a fn()-definition and a fn()-call as seen here.